torch/csrc/api/include/torch/serialize.h

```diff
@@ -72,4 +102,38 @@ void load(Value& value, LoadFromArgs&&... args) {
   archive.load_from(std::forward<LoadFromArgs>(args)...);
   archive >> value;
 }
+
+/// Deserializes the given `tensor_vec` of type `std::vector<torch::Tensor>`.
+///
+/// To perform the serialization, a `serialize::InputArchive` is constructed,
+/// and all arguments after the `value` are forwarded to its `load_from` method.
```

Comment:
this description isn't accurate.  There is no `value` argument for one.

Reply:
Where the arguments are forwarded is also ambiguous (I know this sentence existed beforehand) -- are they forwarded to the `value` or to the `InputArchive`?